src/System.Data.Services.Client/ref/System.Data.Services.Client.depproj

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <OutputType>Library</OutputType>
+    <PackageTargetFramework>net46</PackageTargetFramework>
+    <NuGetTargetMoniker>.NETFramework,Version=v4.6</NuGetTargetMoniker>
+  </PropertyGroup>
+  <ItemGroup>
+    <TargetingPackReference Include="System.Data.Services.Client" />
```

Comment:
I'm OK with this approach for now but we should consider ProjectReference instead.